modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -48,23 +48,20 @@ const SidebarHeader = ({children, title}) => {
 				<ClayLayout.ContentCol>
 					<ClayButtonWithIcon
 						aria-label="Close"
```

Comment:
I just realized that we should translate this text too. We can do it in an other PR if you want.

Reply:
I'll do it! Thxs